backend/src/v5/utils/quota.js

```diff
@@ -15,38 +15,45 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 const { createResponseCode, templates } = require('./responseCodes');
+const { getAllUsersInTeamspace, getSubscriptions } = require('../models/teamspaces');
 const DBHandler = require('../handler/db');
 const config = require('./config');
-const { getSubscriptions } = require('../models/teamspaces');
+const { getInvitationsByTeamspace } = require('../models/invitations');
 const { getTotalSize } = require('../models/fileRefs');
 
 const Quota = {};
 
-const calculateQuota = async (teamspace) => {
+Quota.getQuotaInfo = async (teamspace) => {
 	const subs = await getSubscriptions(teamspace);
-	let quotaSize = 0;
+	let dataSize = 0;
+	let collaborators = config.subscriptions?.basic?.collaborators ?? 0;
 	let hasExpiredQuota = false;
 
+	// eslint-disable-next-line no-loop-func
 	Object.keys(subs).forEach((key) => {
 		// paypal subs have a different schema - and no oen should have an active paypal sub. Skip.
 		if (key !== 'paypal') {
-			const { expiryDate, data } = subs[key];
-
+			const { expiryDate, data, collaborators: subCollaborators } = subs[key];
 			if (expiryDate && expiryDate < Date.now()) {
 				hasExpiredQuota = true;
 			} else {
-				quotaSize += data;
+				dataSize += data;
+				if (collaborators !== 'unlimited') {
+					collaborators = subCollaborators === 'unlimited' ? 'unlimited' : collaborators + subCollaborators;
+				}
 			}
 		}
 	});
 
-	if (hasExpiredQuota && quotaSize === 0) throw templates.licenceExpired;
+	if (hasExpiredQuota && dataSize === 0) throw templates.licenceExpired;
+
+	const basicData = config.subscriptions?.basic?.data;
+	const dataInMegabytes = (dataSize + basicData);
```

Comment:
missing basic collaborator count

Reply:
but Im initializing collaborators to config.subscriptions?.basic?.collaborators